go-controller/pkg/ovn/policy.go

```diff
@@ -740,6 +803,11 @@ func (oc *Controller) deleteNetworkPolicy(policy *knet.NetworkPolicy) {
 	defer np.Unlock()
 
 	oc.destroyNamespacePolicy(np)
+
+	if isLast {
+		deletePortGroup(hashedPortGroup(np.namespace) + "_" + "ingressDefaultDeny")
+		deletePortGroup(hashedPortGroup(np.namespace) + "_" + "egressDefaultDeny")
```

Comment:
might be simpler to just delete these with the namespace, not with the last networkpolicy

Reply:
if we want to delete the default deny with the namespace then shouldn't we also create them with the namespace?

Reply:
however, deleting it on the last policy removal is cleaner as it more symmetric. I.e. we may end up with namespaces w/o policies